src/hooks/useHasLoggedIntoMobileApp.ts

```diff
@@ -0,0 +1,21 @@
+import {useOnyx} from 'react-native-onyx';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+const useHasLoggedIntoMobileApp = () => {
```

Comment:
Can we add a comment explaining what this hook does? Like that it returns whether the user has ever logged into one of the Expensify mobile apps.